docs/source/reference/authenticators.md

```diff
@@ -145,7 +145,7 @@ If such state should be persisted, `.authenticate()` should return a dictionary
 
 ```python
 {
-  'username': 'name',
+  'name': username,
```

Comment:
can you quote `username` again? I think in the rest of this example all values are quoted to make it a valid dict.

Reply:
Without quotes it is more clear that this is a variable and not a string literal.